software/control/gui_hcs.py

```diff
@@ -110,6 +111,7 @@ def __init__(self, is_simulation=False, performance_mode=False, *args, **kwargs)
             self.waitForMicrocontroller()
             if ENABLE_SCAN_GRID:
                 self.multiPointWidgetGrid.init_z()
+            self.multiPointWidget2.init_z()
```

Comment:
Since this isn't related to the performance mode change, it'd be nice to put it in its own pull request.  I think we're aiming for development speed, though, so probably fine for this one!  But in the future having it be as a separate bug fix pull request would be nice.